### ISSUE TITLE
Incompatible with activerecord-import gem

### ISSUE THREAD

**Issue by martin308** [honeycombio/activerecord-honeycomb#9]
When used alongside the [activerecord-import](https://rubygems.org/gems/activerecord-import) gem, database queries are not recorded. This is due to the way that the gem also modifies the `establish_connection` method that we use to hook our custom adapter in. The code below roughly simulates the interaction between the two gems.

```
module ConnectionHandling
  def establish_connection
    puts "establish_connection"
  end
end

class ActiveRecord::Base
  extend ConnectionHandling
end

class ActiveRecord::Base
  class << self
    def establish_connection_rouge_edition
      puts "establish_connection_rouge_edition"
      establish_connection_original
    end

    alias establish_connection_original establish_connection
    alias establish_connection establish_connection_rouge_edition
  end
end

module Beeline
  def establish_connection
    puts "beeline"
  end
end

Honeycomb.extend(Beeline)

Honeycomb.establish_connection

# no beeline 😢
```

**Comment by sethjohnson-flexe** [comment]
The way most instrumentation libraries interact with ActiveRecord is to use ActiveSupport::Notifications.  You can see some examples in the various attempts in the wild to instrument for open tracing.  It should not be necessary to monkey patch ActiveRecord because it already ships with this capability.

**Comment by eanakashima** [comment]
@sethjohnson-flexe thanks, we know there are some deficiencies with the current approach for sure. We have been thinking about some architectural changes that should address them but unfortunately we don't have an ETA on those just yet.

**Comment by martin308** [comment]
@sethjohnson-flexe we have released `v1.0.0` of the beeline now which should solve this issue. Please open up an issue over here https://github.com/honeycombio/beeline-ruby if you still have issues
